OpenGLInterfacing/Cube.cpp

```diff
@@ -0,0 +1,81 @@
+#include "Cube.h"
+
+void Cube::initialize()
+{
+
+}
+void Cube::uninitialize()
+{
+
+}
+
+void Cube::update()
+{
+    angle = angle + 0.1f;
+    if (angle > 360.0f)
+        angle = 0.0f;  
+}
+
+void Cube::display()
+{
+    void drawCube(void);
+    glMatrixMode(GL_MODELVIEW);
+    glLoadIdentity();
+    glTranslatef(0.0f, 0.0f, -5.0f);
```

Comment:
This would show your cube on my triangle right?
So can you move your cube to left - top corner and mine to right - top corner. 
Both figures would be visible.  :)